apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -198,18 +195,18 @@ export function translate24To12HourTime(startTime?: HourMinute, endTime?: HourMi
         return;
     }
 
-    const timeSuffix = endTime.hour >= 12 ? 'PM' : 'AM';
+    const postMeridian = endTime.hour >= 12;
 
-    const formattedStartHour = `${startTime.hour > 12 ? startTime.hour - 12 : startTime.hour}`;
-    const formattedEndHour = `${endTime.hour > 12 ? endTime.hour - 12 : endTime.hour}`;
+    const timeSuffix = postMeridian ? 'PM' : 'AM';
 
-    const formattedStartMinute = `${startTime.minute}`.padStart(2, '0');
-    const formattedEndMinute = `${endTime.minute}`.padStart(2, '0');
+    const formattedStartHour = `${startTime.hour - Number(postMeridian && 12)}`;
+    const formattedEndHour = `${endTime.hour - Number(postMeridian && 12)}`;
```

Comment:
The problem with subtracting 12 based on postMeridian is that when it's 12 PM we don't subtract 12 to get 0 PM, it stays 12 PM.

<img width="862" alt="Screenshot 2023-08-25 at 11 48 54 AM" src="https://github.com/icssc/AntAlmanac/assets/100006999/550f2c48-c326-4b91-b0a4-883b7218e0d8">

I'm not sure what would look cleanest, but  it seems like we would have to have two separate variables, postMeridian and past12 (?).